data/conf/rspamd/local.d/mx_check.conf

```diff
@@ -1,7 +1,8 @@
-timeout = 1.0;
+timeout = 8.0;
 symbol_bad_mx = "MX_INVALID";
 symbol_no_mx = "MX_MISSING";
 symbol_good_mx = "MX_GOOD";
+symbol_implicit_mx = "MX_IMPLICIT";
```

Comment:
Remove this. Mx-check module doesn't know about MX_IMPLICIT.

Reply:
ok, fixed